static/js/home.js

```diff
@@ -19,7 +19,9 @@ var homeJSLocal = {
   searchBox: undefined,
   drawingManager: undefined,
   clearSelectAreaButton: undefined,
-  detailViewState: false
+  detailViewState: false,
+  waitingDialog: waitingDialog,
+  globalInit: false
```

Comment:
This variable is to notify that initialization has been finished for the page. It is done when the page has been loaded. This is required because , waiting dialog needs to know when initial thing has been done, Otherwise it may run to into thread issues and system hangs.